src/index.ts

```diff
@@ -12,19 +12,17 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 //
-// ** This file is automatically generated by gapic-generator-typescript. **
-// ** https://github.com/googleapis/gapic-generator-typescript **
+// ** This file is automatically generated by synthtool. **
```

Comment:
@bcoe This change is actually coming from the post-processor, and it's incorrect.  How do we fix it?